Source/Core/DolphinQt/Debugger/CodeDiffDialog.cpp

```diff
@@ -0,0 +1,515 @@
+// Copyright 2022 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "DolphinQt/Debugger/CodeDiffDialog.h"
+
+#include <QHBoxLayout>
+#include <QLabel>
+#include <QMenu>
+#include <QPushButton>
+#include <QTableWidget>
+#include <QVBoxLayout>
+#include <string>
+#include <vector>
+#include "Common/StringUtil.h"
+#include "Core/Core.h"
+#include "Core/Debugger/PPCDebugInterface.h"
+#include "Core/HW/CPU.h"
+#include "Core/PowerPC/JitInterface.h"
+#include "Core/PowerPC/MMU.h"
+#include "Core/PowerPC/PPCSymbolDB.h"
+#include "Core/PowerPC/PowerPC.h"
+#include "Core/PowerPC/Profiler.h"
+#include "DolphinQt/Debugger/CodeWidget.h"
+#include "DolphinQt/Host.h"
+#include "DolphinQt/QtUtils/ModalMessageBox.h"
+#include "DolphinQt/Settings.h"

```

Comment:
Can you split these into four groups separated by empty lines? STL stuff first, ideally. You also need `<algorithm>` for `std::lower_bound`.

Reply:
Yes, CodeDiffDialog.h should be first, on its own line.
And yes, nothing in the header itself requires `algorithm` so it should not be included there.